contrib_bots/lib/CommuteBot/CommuteBot.py

```diff
@@ -0,0 +1,150 @@
+    Optional Inputs:
+        Units (Metric or imperial) (units=metric)
+        Restrictions (Accepted inputs:tolls, highways, ferries, indoor) (avoid=tolls)
+        Departure Time (Seconds After Midnight January 1, 1970 UTC) (departure_time=now)
+        Arrival Time (Seconds After Midnight January 1, 1970 UTC) (arrival_time=732283981023)
+        Language (Available languages available here: https://developers.google.com/maps/faq#languagesupport) (lan=fr)
+
+    Sample request:
+        @commute origin=Chicago,IL,USA destination=New+York,NY,USA mode=driving language=fr
+
+    Please note:
+        Fare information can be derived, though is solely dependent on the
+         availability of the information released by public transport operators
+        Duration in traffic can only be derived if a departure time is set.
+        If a location has spaces in its name, please use a + symbol in the place of the space/s
+        A departure time and a arrival time can not be inputted at the same time
+        To add more than 1 input for a category,
+        eg. more than 1 destinations, use |, eg. destinations=Trump+Tower|The+White+House
+        No spaces within addresses.
+                            '''
+            self.send_info(message, help_info, client)
+        else:
+            para = {}
+            # converts input parameters for api into a dictionary structure for more aethestic coding
+            for item in content_list:
+                # enables key value pair
+                org = item.split('=')
+
+                # ensures that invalid inputs are not entered into url request
+                if len(org) != 2:
+                    continue
+
+                key = org[0]
+                value = org[1]
+                para.update({key: value})
+
+            # adds API Authentication Key to url request
+            para.update({'key': 'AIzaSyCGrx2FIMDVvfjYGgIjN9ocKyd5cfOiH-M'})
+
+            # sends url api request and converts response into json format
+            r = requests.get('https://maps.googleapis.com/maps/api/distancematrix/json', params=para)
+            rjson = r.json()
+
+            # determines if user has valid inputs
+            try:
+                not_found = (rjson["rows"][0]["elements"][0]["status"] == "NOT_FOUND")
+                invalid_request = (rjson["status"] == "INVALID_REQUEST")
+                no_result = (rjson["rows"][0]["elements"][0]["status"] == "ZERO_RESULTS")
+
+                if no_results:
+                    self.send_info(message, "Zero results\nIf stuck, try '@commute help'", client)
+                    return None
+
+                elif not_found or invalid_request:
+                    raise IndexError
+            except IndexError:
+                self.send_info(message,
+                               "Invalid input, please input as per instructions.\nIf stuck, try '@commute help'", client)
```

Comment:
Missing full stop, also, this line is a tad long, could you put `client` on a separate line?